libzkbob-rs/src/pools.rs

```diff
@@ -14,62 +14,66 @@ pub const GENERIC_ADDRESS_PREFIX: &str = "zkbob";
 // It used to support multipool shielded addresses
 #[derive(Copy, Clone, PartialEq, Debug)]
 pub enum Pool {
-    PolygonBOB,
-    OptimismBOB,
-    OptimismETH,
     PolygonUSDC,
+    OptimismUSDC,
+    OptimismETH,
+    TronUSDT,
     SepoliaBOB,
     GoerliBOB,
-    GoerliOptimismBOB,
+    GoerliOptimismUSDC,
     GoerliETH,
     GoerliUSDC,
+    ShastaUSDT,
 }
 
 impl Pool {
     pub fn from_pool_id(pool_id: u32) -> Option<Pool> {
         match pool_id {
-            0x000000 => Some(Pool::PolygonBOB),
```

Comment:
Wdyt about adding these parameters to a pool config file which is already used by console and front end? 
Hardcoding it seems suboptimal

Reply:
Do not agree about the config file. In that case we should add pool prefix as well as `pool_id` to the config (it's using for checksum calculation). Imagine someone make a typo in the config file. It'll cause the addresses generated from such application will invalid in the 'correct' application. Are you sure we should move such critical map to the external config? I believe the map must remain hardcoded anyway. Moving this map from the rust library to the JS lib looks more reasonable. But I propose to make it after the current review within a separated task to reduce time to release. Wdyt?

Reply:
I agree that it makes sense to refactor this part. I guess it is fine to hardcode these values if adding a new pool will require some trivial changes. Currently, to add a new pool we need to update enum and 5 functions which is definitely not optimal.

I also think that it is better to refactor this code in a separate PR.